pyproject.toml

```
@@ -76,5 +76,5 @@ authorized_licenses = [
 
 
 [build-system]
-requires = ["poetry>=1.0.0"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
```

Same as https://github.com/fedora-infra/noggin/pull/1153#discussion_r1123559117

```suggestion
requires = ["poetry-core"]
```